src/chat/components/InputToolbar.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import {
+  View,
+  StyleSheet,
+  ScrollView,
+  TouchableOpacity,
+  Image,
+  StyleProp,
+  ViewStyle,
+  ImageStyle,
+} from 'react-native';
+import {
+  Composer,
+  InputToolbarProps,
+  SendProps,
+} from 'react-native-gifted-chat';
+
+interface IInputToolbar extends InputToolbarProps<any>, SendProps<any> {
+  isShowCamera?: boolean;
+  isShowPhotoGallery?: boolean;
+  togglePhotoGallery?: () => void;
+  containerStyle?: StyleProp<ViewStyle>;
+  composeWrapperStyle?: StyleProp<ViewStyle>;
+  composerTextInputStyle?: StyleProp<ViewStyle>;
+  customViewStyle?: StyleProp<ViewStyle>;
+  iconCamera?: any;
+  iconGallery?: any;
+  iconSend?: any;
+  iconStyle?: StyleProp<ImageStyle>;
+  iconSize?: number;
+  iconMargin?: number;
+}
+
+export const PressableIcon: React.FC<{
+  icon: any;
+  style?: StyleProp<ViewStyle>;
+  iconStyle?: StyleProp<ImageStyle>;
+  onPress?: () => void;
+  resizeMode?: 'contain' | 'cover' | 'stretch' | 'repeat' | 'center';
+  disabled?: boolean;
+  hitSlop?: { top?: number; right?: number; bottom?: number; left?: number };
+  size?: number;
+}> = ({
+  icon,
+  style,
+  iconStyle,
+  onPress,
+  resizeMode = 'contain',
+  disabled,
+  hitSlop,
+  size = 18,
+}) => (
+  <TouchableOpacity
+    disabled={disabled}
+    hitSlop={{ top: 10, right: 10, bottom: 10, left: 10, ...hitSlop }}
```

Comment:
create constant for hitSlop

Reply:
Yes, I created a constant,  please check it